src/printer.js

```diff
@@ -16,10 +16,23 @@ const {
   breakParent,
 } = require("prettier").doc.builders;
 const { willBreak } = require("prettier").doc.utils;
-const { makeString, isNextLineEmpty } = require("prettier").util;
+const {
+  makeString,
+  isNextLineEmpty: _isNextLineEmpty,
+} = require("prettier").util;
 const { isValidIdentifier, isExpression } = require("./util");
 const { printDanglingComments, isDanglingComment } = require("./comments");
 
+const prettierVersion = require("prettier").version;
+
+function lookupIfPrettier2(options, prop) {
+  return parseInt(prettierVersion[0]) > 1 ? options[prop] : options;
```

Comment:
This way, the version check won’t break on **Prettier 10** – **Prettier 19**:
```suggestion
const [prettierVersionMajor] = require("prettier").version.split(".", 1);

function lookupIfPrettier2(options, prop) {
  return parseInt(prettierVersionMajor) > 1 ? options[prop] : options;
```